conf/nginx/yunohost_api.conf.inc

```diff
@@ -23,3 +23,23 @@ location = /yunohost/api/error/502 {
     add_header Content-Type text/plain;
     internal;
 }
+
+location /yunohost/portalapi/ {
+    proxy_read_timeout 3600s;
```

Comment:
That's really high timeout for this central part of the system. Of course it should never timeout but just in case receiving error after 5s is better than waiting 1h